actionmailbox/test/controllers/ingresses/mandrill/inbound_emails_controller_test.rb

```diff
@@ -10,6 +10,13 @@ class ActionMailbox::Ingresses::Mandrill::InboundEmailsControllerTest < ActionDi
     @events = JSON.generate([{ event: "inbound", msg: { raw_msg: file_fixture("../files/welcome.eml").read } }])
   end
 
+  test "verifying existence of mandrill inbound route" do
+    # the first request has a default INGRESS_API_TOKEN of "test-webhook"
+    get rails_mandrill_inbound_emails_url,
```

Comment:
This should be `head`, right?

My reading of the documentation is that the HEAD request won't be signed at all - it's only the fallback POST that uses the "test-webhook" key.

Would you be able to verify this behaviour with a real Mandrill account?

Reply:
Indeed, looks like it. I'll make the change.

It also appears as if some of the tests in railties break as a result of this change:
`
railties/test/commands/routes_test.rb:166
railties/test/commands/routes_test.rb:202
activemodel/test/application/rake/routes_test.rb:20
` 

Should I update those tests so that they pass?

There are other tests that fail and I can't quite figure out why...
Should I continue to attempt to debug them?

Reply:
Yes, please do update those tests to reflect the new route added here. Ideally those tests would be more resilient to incidental changes... but the easiest path here is to play along 🙂

The other failures are against the latest nightly snapshot of Ruby, and won't cause the built to fail on their own - you can see them also failing on the latest (passing) master build here:

https://buildkite.com/rails/rails/builds/67787

Reply:
Updated the tests. I *think* all of the tests that this change broke are now passing. Lets see what the buildkite comes back with. Regarding the conflicts below, I assume the maintainers can fix those problems?

Reply:
Looks like the tests are indeed passing now! ⭐️

We can fix the conflicts when merging if necessary, but if you're able to, rebasing your branch and fixing them yourself would help us out:

https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#update-your-branch